comments.txt

```diff
@@ -0,0 +1,22 @@
+ comment_id | votes |    author     | article_id |     created_at      
+------------+-------+---------------+------------+---------------------
+          1 |    16 | butter_bridge |          9 | 2020-04-06 13:17:00
+          2 |    14 | butter_bridge |          1 | 2020-10-31 03:03:00
+          3 |   100 | icellusedkars |          1 | 2020-03-01 01:13:00
+          4 |  -100 | icellusedkars |          1 | 2020-02-23 12:01:00
+          5 |     0 | icellusedkars |          1 | 2020-11-03 21:00:00
+          6 |     0 | icellusedkars |          1 | 2020-04-11 22:02:00
+          7 |     0 | icellusedkars |          1 | 2020-05-15 21:19:00
+          8 |     0 | icellusedkars |          1 | 2020-04-14 21:19:00
+          9 |     0 | icellusedkars |          1 | 2020-01-01 03:08:00
+         10 |     0 | icellusedkars |          3 | 2020-06-20 08:24:00
+         11 |     0 | icellusedkars |          3 | 2020-09-20 00:10:00
+         12 |     0 | icellusedkars |          1 | 2020-03-02 07:10:00
+         13 |     0 | icellusedkars |          1 | 2020-06-15 11:25:00
+         14 |    16 | icellusedkars |          5 | 2020-06-09 06:00:00
+         15 |     1 | butter_bridge |          5 | 2020-11-24 00:08:00
+         16 |     1 | butter_bridge |          6 | 2020-10-11 16:23:00
+         17 |    20 | icellusedkars |          9 | 2020-03-14 17:02:00
+         18 |    16 | butter_bridge |          1 | 2020-07-21 01:20:00
+(18 rows)
+
```

Comment:
I would gitignore these files if they are just there for you to test out what the data looks like!